### PR TITLE
Walkthrough

### PR DESCRIPTION
**Pull request by RyanProffitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,115 @@
 # stellar
 
-The `stellar` command line interface (CLI) is an __experimental__ tool primarily used by the StellarStation development team
-to perform internal end-to-end tests when developing the [StellarStation API](https://github.com/infostellarinc/stellarstation-api).
+The `stellar` command line interface (CLI) is an __experimental__ tool. Support is provided on a "best effort" basis. It is strongly recommended that users develop their own client-side software to interface with the StellarStation API.
 
-For best results, it is strongly recommended that users develop their own client interfaces which communicate directly
-with the defined StellarStation API, rather than through the `stellar` CLI tool.
+See the [documentation](/docs/stellar.md) for more details about commands.
```
Comment (JamieCrisman) [inline README.md]: In the CLI, you can use:

```sh
stellar --help
```
to see documentation and commands that are available in your current version.

File: README.md [hunk 2]
```diff
@@ -1,72 +1,115 @@
...
-```bash
-$ stellar -h
-```
+### Getting a list of Available Passes
 
-If you see anything is unclear, feel free to file an issue or send a pull request.
+We can see a list of available passes now that the TLE is set.
 
-## Authentication
+Execute `stellar sat list-passes [STELLARSTATION SATELLITE ID HERE] --verbose`.
```
Comment (JamieCrisman) [inline README.md]: `stellar satellite list-passes`

File: README.md [hunk 3]
```diff
@@ -1,72 +1,115 @@
...
-```bash
-$ stellar auth activate-api-key path/to/stellarstation-private-key.json
-```
+### Reserving a Plan
 
-All commands after that will be authenticated using that key.
+We can make a reservation now that we have the list of passes. Try reserving the last available pass in the list using it's `reservationToken`.
 
-### interactive-plan: Why are things misaligning? Why are borders at the wrong widths?
+Execute `stellar sat reserve-pass [reservationToken]`.
```
Comment (JamieCrisman) [inline README.md]: `stellar satellite`

File: README.md [hunk 4]
```diff
@@ -1,72 +1,115 @@
...
 
-```bash
-# Go users
-$ go build
-```
+### Why are things misaligning and borders at the wrong widths when I use `interactive-plan`?
+This is most likely due to your locale and encoding, particularly with regard to Chinese, Japanese, and Korean (for example, `zh_CN.UTF-8` or `ja_JP.UTF-8`). The most direct way to fix this is to set `RUNEWIDTH_EASTASIAN=0` in your environment. For more details see https://github.com/charmbracelet/lipgloss/issues/40.
 
-### Linting
+### What version of Go does Infostellar use?
```
Comment (JamieCrisman) [inline README.md]: I'd keep the releaser/linting stuff under a development section

File: README.md (hunk 2 above)
Comment (RyanProffitt) [reply README.md:72]: To clarify, you want me to expand the name of 'sat' to 'satellite'?

File: README.md [hunk 5]
```diff
@@ -1,72 +1,117 @@
...
 
-```bash
-# Go users
-$ go build
-```
+Now that you have completed authentication, we can update the TLE. This walkthrough assumes an Infostellar support engineer has configured a test ground station for your account below 51 degrees latitude.
+
+Execute `stellar satellite add-tle [STELLARSTATION SATELLITE ID HERE] "[TLE LINE 1]" "[TLE LINE 2]"`. Try the [Space Station](https://celestrak.org/NORAD/elements/gp.php?CATNR=25544) as a good temporary TLE for now.
+
+Now that you have provided a new TLE, wait several seconds, and check that the TLE propagated to StellarStation.
```
Comment (rockets3600) [inline README.md]: There's no need to wait, get-tle will return the new tle immediately.

**Review by rockets3600 — APPROVED** [review]

**Pull request merged**